assets/dev/scss/global/_dialog-confirm.scss

```diff
@@ -22,11 +22,11 @@ $e-dialog-confirm-space: 20px;
 		&:after {
 			content: '';
 			display: block;
-			border-bottom: var(--e-a-border);
-			padding-bottom: 10px;
-			margin-bottom: 10px;
-			margin-right: -$e-dialog-confirm-space;
-			margin-left: -$e-dialog-confirm-space;
+			border-block-end: var(--e-a-border);
+			padding-block-end: 10px;
+			margin-block-end: 10px;
+			margin-inline-start: -$e-dialog-confirm-space;
+			margin-inline-end: -$e-dialog-confirm-space;
```

Comment:
```suggestion
			margin-inline: -$e-dialog-confirm-space;
```

Reply:
I know, I thought about it for a long time. But I decided not to use the shorthand because of **Safari**.

Both `margin-inline-start` and `margin-inline-end` are supported since **Safari 12.1** (released on 2019) while `margin-inline` shorthand is supported since **Safari 14.1** (released on 2021). For now, no shorthand properties.